dependencies/synapse/modules/core/src/main/java/org/apache/synapse/endpoints/AbstractEndpoint.java

```diff
@@ -191,9 +191,9 @@ public void setName(String endpointName) {
                 return;
             }*/
             metricsMBean = new EndpointView(endpointName, this);
-            if(metricsMBean != null) {
-                metricsMBean.destroy();
-            }
+//            if(metricsMBean != null) {
```

Comment:
This commented code has been removed in the commit dc2e4245031771730bc811c0dd6e4ada87ee2aa9